app/api/events.py

```diff
@@ -64,23 +111,30 @@ class Meta:
                            many=True,
                            type_='sponsor')
 
+    user = Relationship(attribute='user',
+                        self_view='v1.event_user',
+                        self_view_kwargs={'id': '<id>'},
+                        related_view='v1.user_detail',
+                        related_view_kwargs={'id': '<id>'},
+                        schema='UserSchema',
+                        type_='user')
+
 
 class EventList(ResourceList):
-    decorators = (jwt_required, )
-    schema = EventSchema
-    data_layer = {'session': db.session,
-                  'model': Event}
 
+    def before_create_object(self, data, view_kwargs):
+        data['starts_at'] = data['starts_at'].replace(tzinfo=None)
+        data['ends_at'] = data['ends_at'].replace(tzinfo=None)
```

Comment:
@LuD1161 Yes, you are saving it correctly, it is not timezone aware. Saving to database is not problem.

 Right now what is happening is you should make it timezone aware while getting a GET response.
So now while I try to GET for an event (say starts_at 2017/06/02 10:00 and ends at starts_at 2017/06/10 18:00 in timezone Asia/Kolkata), I should get a timezone aware response as "2017-06-02T10:00+03.00" and "2017-060-10T18:00+03.00". however I am getting "2017-06-10T18:00+00.00" which is a fixed UTC format.

I think this is what is wanted right now. Correct me if I am wrong.